libr/include/r_endian.h

```diff
@@ -0,0 +1,98 @@
+/* radare - LGPL - Copyright 2016 - damo22 */
+
+#if LIL_ENDIAN
+#define CFG_BIGENDIANHOST "false"
+#else
+#define CFG_BIGENDIANHOST "true"
+#endif
+
+/* this default will change when the endianness is set */
+#define CFG_BIGENDIANTARGET CFG_BIGENDIANHOST
+
+#define BIGENDIAN(x)                 \
```

Comment:
dont do that :( every plugin have a callback that says if its big or little. hardcoding this here is ugly, slow and a bad idea